src/data.js

```diff
@@ -22,6 +22,25 @@ module.exports = [
       'Platforms',
     ],
   },
+  {
+    name: 'Erik Thiart',
+    description:
+      'Full Stack Developer, I am a Swiss Army knife.',
+    url: 'https://erikthiart.com/uses/',
```

Comment:
This just redirects to the homepage, could you create a uses page?

Reply:
Oopsie... I thought it was just robots adding us... Hehe, caught red-handed.

I have added a uses page https://erikthiart.com/uses/index.php and removed the .htaccess page, check if it works for you now?

Reply:
Cool works now